ccmlib/scylla_repository.py

```diff
@@ -15,12 +7,15 @@
 import sys
 import glob
 import urllib
-
 import hashlib
+import logging
+import random
+import time
```

Comment:
i assume that you want to reorder the `import` statements. but in what order? in alphabetical order ? if yes, then there are still some other `import`s are not ordered.

Reply:
The grouping of them is more important to me than the alphabetic order or any other order.

I.e. builtin first, third party, in package import

Reply:
okay, i see. makes more sense now.